flight/flight-core/src/main/java/org/apache/arrow/flight/FlightServer.java

```diff
@@ -552,5 +570,20 @@ public Builder producer(FlightProducer producer) {
       this.producer = Preconditions.checkNotNull(producer);
       return this;
     }
+
+    private void prepareTlsSettings() throws IOException {
```

Comment:
The way this is written, if you set the File version then the deprecated version never takes effect, even if you set it afterwards. I suppose that's OK given the deprecation.